Codes/Driver/NJH/main.cpp

```diff
@@ -0,0 +1,67 @@
+﻿#include "ISP.h"
+
+const String windowName = "Sleep Detection";
+const String subwindowName = "Preprocessing";
+const int log_var = 1.1; //log var 크기만큼 맵핑
+const float gamma_var = 2; //커질수록 어두워짐
```

Comment:
혹시 이 부분 값이 서로 바뀐 걸까요? 데이터 타입 확인 부탁드립니다!
```suggestion
const int log_var = 2; //log var 크기만큼 맵핑
const float gamma_var = 1.1; //커질수록 어두워짐
```